usageReports/rds/prepare_response.go

```diff
@@ -0,0 +1,159 @@
+//   Copyright 2018 MSolution.IO
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package rds
+
+import (
+	"strings"
+	"context"
+	"net/http"
+	"encoding/json"
+
+	"gopkg.in/olivere/elastic.v5"
+
+	"github.com/trackit/trackit-server/aws/usageReports/rds"
+)
+
+type (
+
+	// Structure that allow to parse ES response for costs
+	ResponseCost struct {
+		Accounts struct {
+			Buckets []struct {
+				Key string `json:"key"`
+				Instances struct {
+					Buckets []struct {
+						Key string `json:"key"`
+						Cost struct {
+							Value float64 `json:"value"`
+						} `json:"cost"`
+					} `json:"buckets"`
+				} `json:"instances"`
+			} `json:"buckets"`
+		} `json:"accounts"`
+	}
+
+	// Structure that allow to parse ES response for RDS usage report
+	ResponseRds struct {
+		TopReports struct {
+			Buckets []struct {
+				TopReportsHits struct {
+					Hits struct {
+						Hits []struct {
+							Source rds.Report `json:"_source"`
+						} `json:"hits"`
+					} `json:"hits"`
+				} `json:"top_reports_hits"`
+			} `json:"buckets"`
+		} `json:"top_reports"`
+	}
+)
+
+// addCostToReport adds cost for each instance based on billing data
+func addCostToReport(report rds.Report, costs ResponseCost) (rds.Report) {
+	for _, accounts := range costs.Accounts.Buckets {
+		if accounts.Key != report.Account {
+			continue
+		}
+		for _, instance := range accounts.Instances.Buckets {
+			for i := range report.Instances {
+				if report.Instances[i].CostDetail == nil {
+					report.Instances[i].CostDetail = make(map[string]float64, 0)
+				}
+				split := strings.Split(instance.Key, ":")
+				if len(split) == 7 && split[6] == report.Instances[i].DBInstanceIdentifier {
+					report.Instances[i].Cost += instance.Cost.Value
+					report.Instances[i].CostDetail["instance"] += instance.Cost.Value
+				}
+			}
+		}
+	}
+	return report
+}
+
+// prepareResponseRdsDaily parses the results from elasticsearch and returns the RDS report
+func prepareResponseRdsDaily(ctx context.Context, resRds *elastic.SearchResult, resCost *elastic.SearchResult) ([]rds.Report, error) {
+	parsedRds  := ResponseRds{}
```

Comment:
There is a two spaces instead of one here, which tells me you didn't run `go fmt` on your sources. Please do, as it allows to have the same indentation for every file in the repository.